src/components/_uswds-form.njk

```diff
@@ -0,0 +1,8 @@
+{% extends '_uswds.njk' %}
+{% block body %}
+  <div class="grid-container padding-x-2 tablet-padding-x-4 padding-y-4">
+    <form class="usa-form">
+      {{ yield | safe }}
```

Comment:
Added a form template to prevent having to wrap form controls in a `form` element